### PR TITLE
Update sig testing charter.md

### PR DESCRIPTION
**Pull request by aojea**
Add the definition of the testing strategy and guidelines to the charter and clarify that the testing is a responsability of each SIG

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: sig-testing/charter.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ away project toil. We focus on creating and running tools and infrastructure
 that make it easier for the community to write and run tests, and to
 contribute, analyze and act upon test results.
 
+We define the overall testing standards, strategies and style guidelines for
+the project, although, each SIG is responsible of the implementation and
+interpretation of these guidelines.
```
Comment (pohly) [inline sig-testing/charter.md:16]: Slight wording tweak:

```suggestion
the project, although, each SIG is responsible for the implementation and
interpretation of these guidelines in the context of their work.
```

I'm still a bit uneasy about the "interpretation of these guidelines": if the guidelines need interpretation, then they were not clear enough. I think what this says is "need to decide to which extend they want to follow those guidelines".

So how about this:
```suggestion
the project. Each SIG is responsible for determining how those
guidelines apply to their work and implementing those parts
which make sense.
```

**Review by michelle192837 — APPROVED** [review]
Thanks! Looks good to me (w/ Patrick's comment), so LGTM'ing w/ hold.

/hold

File: sig-testing/charter.md (hunk 1 above)
Comment (BenTheElder) [reply sig-testing/charter.md:16]: I agree.

**Comment by BenTheElder** [comment]
+1 in concept, also +1 for the review suggestions from Patrick :-)

**Comment by aojea** [comment]
applied Patrick suggestions, agree with it

**Comment by pohly** [comment]
The revised version in [a27c81b](https://github.com/kubernetes/community/pull/7730/commits/a27c81b2302ec4de4b576e291e705234894b7b20) LGTM.

**Comment by aojea** [comment]
> The revised version in [a27c81b](https://github.com/kubernetes/community/pull/7730/commits/a27c81b2302ec4de4b576e291e705234894b7b20) LGTM.

/hold cancel

@pohly @michelle192837 are we good for lgtm?

**Comment by michelle192837** [comment]
Belated LGTM, thanks y'all!

**Pull request merged**